data/json/effects.json

```
@@ -1983,14 +1984,14 @@
     "name": [ "Mild hypovolemic shock", "Moderate hypovolemic shock", "Advanced hypovolemic shock", "Severe hypovolemic shock" ],
     "desc": [
       "You've lost some blood and look somewhat pale.",
-      "You've lost large amount of blood, and you're not feeling well.",
+      "You've lost a large amount of blood, and you're not feeling well.",
       "You've lost an awful lot of blood, and your condition is severe.  Seek medical attention.",
       "You've lost tremendous amount of blood, and you're standing on death's door.  Transfusion might save you."
     ],
     "apply_message": "You have lost lot of blood, and your condition worsens.",
     "remove_message": "You are no longer in shock.",
     "decay_messages": [
-      [ "Your blood level increase, and you feel better, but still look pale.", "good" ],
+      [ "Your blood levels increase, and you feel better; but still, you look pale.", "good" ],
```

```suggestion
      [ "Your blood volume increases, and you feel better; but still, you look pale.", "good" ],
```
"Levels" is what I would use to describe how much of something is in the blood - blood calcium levels, blood lead levels...

---

Yeah, you're right. I didn't find it particularly wrong on a first pass, but now I agree with you.